config/webpack.config.js

```diff
@@ -33,7 +44,80 @@ const globalEntryFiles = {
   ],
 };
 
-const configGenerator = (buildOptions, apps) => {
+/**
+ * Get a list of all the entry points.
+ *
+ * If the `entry` CLI argument is passed, only the specified
+ * application entries are built.
+ */
+function getEntryPoints(entry) {
+  const manifests = getAppManifests(path.join(__dirname, '../'));
```

Comment:
Due to your refactor do we need to pass it an arg now?

Reply:
Oh, good call. :male_detective: